src/zope/interface/_zope_interface_coptimizations.c

```diff
@@ -776,7 +776,7 @@ typedef struct {
 
 static struct PyMethodDef ib_methods[] = {
   {"__adapt__", (PyCFunction)__adapt__, METH_O,
-   "Adapt an object to the reciever"},
+   "Adapt an object to the receiver"},
```

Comment:
@jamadden or @d-maurer - I am not sure whether this change could have an undwanted side-effect. Could any of you please give a final review on this PR? Thansk!